.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Summary
+
+*Briefly describe the issue and please DELETE the non applicable headings below. *
+
+## System information
+
+*Please specify the Kolibri version you were using and your operating system!*
+
+ - Version: ?
+ - Operating system: ?
+ - Browser: ?
+
+## Traceback or relevant snippet from server.log
```

Comment:
Modify to:
`## Traceback or relevant snippet from server.log or browser console`